lib/profileCache.js

```diff
@@ -71,7 +71,7 @@ module.exports = function profileCache(server, options) {
   }, {
     cache: {
       expiresIn: options.expiresIn,
-      generateTimeout: options.generateTimeout
+      generateTimeout: options.generateTimeout || 100
```

Comment:
I think we change that in the tests instead and should not do this here

Reply:
it was throwing error here 'generateTimeout is required ` so i added it temporarily